include/view.h

```diff
@@ -147,6 +148,8 @@ class view {
 	void cancel_input(std::shared_ptr<formaction> fa);
 	void delete_word(std::shared_ptr<formaction> fa);
 
+	cache* rsscache;
```

Comment:
On a principle that all new code should be top-notch, please initialize that pointer to `nullptr`.